From 838672974ec35c9e9eea4f67a2e7206f5cf87c22 Mon Sep 17 00:00:00 2001
From: David Henningsson <david.henningsson@canonical.com>
Date: Mon, 15 Aug 2011 12:05:47 +0000
Subject: alsa-mixer: Mute IEC958 optical raw for several Audigy models

This switch needs to be off for analog output to work on several
Audigy cards.

Thanks to Daniel T Chen for the patch.

BugLink: http://bugs.launchpad.net/bugs/408370
Signed-off-by: David Henningsson <david.henningsson@canonical.com>
---
diff --git a/src/modules/alsa/mixer/paths/analog-output.conf.common b/src/modules/alsa/mixer/paths/analog-output.conf.common
index ccaa494..db6b473 100644
--- a/src/modules/alsa/mixer/paths/analog-output.conf.common
+++ b/src/modules/alsa/mixer/paths/analog-output.conf.common
@@ -129,6 +129,9 @@ priority = 0
 name = output-bass-boost-off
 priority = 10
 
+[Element IEC958 Optical Raw]
+switch = off
+
 ;;; 'Analog Output'
 
 [Element Analog Output]
--
cgit v0.9.0.2-2-gbebe
